frontend/src/app/main/data/team.cljs

```diff
@@ -118,8 +119,10 @@
       (let [team-id (:current-team-id state)
             teams   (get state :teams)
             team    (get teams team-id)]
-        (rx/of (set-current-team team)
-               (fetch-members))))))
+        (if (not team)
+          (rx/of (ex/raise :type :authentication))
```

Comment:
`rx/of` here is useless,. because ex/raise throws an exception.

I think this should be like `(when-not team (rx/raise...))`

Reply:
You're right. I'll chang it to `(rx/throw (ex/error...`